libs/solid/skeleton/src/lib/components/info/info.component.html

```diff
@@ -1,11 +1,14 @@
-<div class="container">
+<div class="container" *ngIf="!isFetching">
```

Comment:
Was drückt der Name aus? Wenn die Messages geladen werden?
Dann ist `isLoading` oder `messagesLoading` vielleicht besser.
Wir haben im [Quiz auch quizLoading](https://github.com/zentrumnawi/solid-frontend/blob/dev/libs/solid/quiz/src/lib/components/start/start.component.html#L31) verwendet (nicht als Variable, sondern nur als anchor, aber das ist ja vom sinn her vergleichbar). In den Profiles haben wir auch [isLoading](https://github.com/zentrumnawi/solid-frontend/blob/dev/libs/solid/profile/src/lib/components/base/base.component.ts#L96) verwendet.

Reply:
(da könnte man natürlich auch für die Konsistenz `profilesLoading` verwenden)

Reply:
Alles klar!